db/database.go

```diff
@@ -906,6 +931,24 @@ func VacuumAttachments(bucket base.Bucket) (int, error) {
 	return 0, base.HTTPErrorf(http.StatusNotImplemented, "Vacuum is temporarily out of order")
 }
 
+//backgroundTask runs task at the specified time interval in its own goroutine until the changeCache is stopped
+func (context *DatabaseContext) backgroundTask(name string, task BackgroundTaskFunc, interval time.Duration) {
```

Comment:
Would be ideal to consolidate this with the existing changeCache backgroundTask.